plotting_tools/PlotHelpers.h

```diff
@@ -0,0 +1,222 @@
+TH1D* get_hist_with_same_width_bins(TH1D* input_hist, string input_hist_name, vector<int> lxy_bins, float rebin, float int_lumi, float cross_sec, float BR, float N_tot)
```

Comment:
`int_lumi` being a float... :D Don't be afraid to give a bit longer, but non-ambiguous names. The same is true for `cross_sec`, `N_tot`, but especially `BR`!